### ISSUE TITLE
PHP error in plugin?

### ISSUE THREAD

**Issue by shawaj** [angelleye/paypal-woocommerce#738]
Getting lots of the following error in logs:

[Wed May 17 12:10:57.663580 2017] [fcgid:warn] [pid 23681:tid 1407xxxxxxxx08] [client xxx.xxx.xxx.xxx:xxxxx] mod_fcgid: stderr: PHP Warning:  Missing argument 1 for WC_Checkout::create_order(), called in /srv/http/vhosts/site.com/httpdocs/wp-content/plugins/paypal-for-woocommerce/classes/wc-gateway-paypal-express-angelleye.php on line 1200 and defined in /srv/http/vhosts/site.com/httpdocs/wp-content/plugins/woocommerce/includes/class-wc-checkout.php on line 269, referer: https://www.site.com/checkout/review-order/?utm_nooverride=1&pp_action=revieworder&token=EC-4Jxxxxxxxxx251&PayerID=WxxxxxxxxxN72

**Comment by kcppdevelopers** [comment]
@shawaj  Can you please provide WooCommerce system report? https://www.screencast.com/t/HtEMGF6Ga0N

**Comment by shawaj** [comment]
```
### WordPress Environment ###

Home URL: https://www.site.com
Site URL: https://www.site.com
WC Version: 3.0.7
Log Directory Writable: ✔
WP Version: 4.7.5
WP Multisite: –
WP Memory Limit: 384 MB
WP Debug Mode: –
WP Cron: –
Language: en_US

### Server Environment ###

Server Info: Apache/2.4.10 (Debian)
PHP Version: 5.6.30-0+deb8u1
PHP Post Max Size: 8 MB
PHP Time Limit: 600
PHP Max Input Vars: 1000
cURL Version: 7.38.0
OpenSSL/1.0.1t

SUHOSIN Installed: –
MySQL Version: ❌ 5.5.55 - We recommend a minimum MySQL version of 5.6. See: WordPress requirements
Max Upload Size: 8 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.0.7
WC Database Prefix: wp_
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (75) ###

VaultPress: by Automattic – 1.8.9
WooCommerce Blacklister: by Aelia (Diego Zanella) – 0.8.6.150612
AffiliateWP - PayPal Payouts: by Pippin Williamson and Andrew Munro – 1.1.7
AffiliateWP: by AffiliateWP
LLC – 2.0.9

Akismet Anti-Spam: by Automattic – 3.3.2
BitPay for WooCommerce: by bitpay – 2.2.9
Captcha by BestWebSoft: by BestWebSoft – 4.3.0
Co-Authors Plus: by Mohammad Jangda
Daniel Bachhuber
Automattic – 3.2.2

WooCommerce Gift Certificates Pro: by IgniteWoo.com – 3.5.32
IgniteWoo Updater: by IgniteWoo.com – 2.1.9
iThemes Security Pro: by iThemes – 3.7.5
Jetpack by WordPress.com: by Automattic – 4.9
KingSumo Giveaways: by KingSumo – 1.5.2
Learn: by Mario Tadic – 1.0.1
Limit Login Attempts: by Johan Eenfeldt – 1.7.1
MailChimp for WooCommerce: by MailChimp – 2.0.0
Manage Distributors: by techEsthete( Rashid Ali) –
Page Links To: by Mark Jaquith – 2.9.9
PayPal for WooCommerce: by Angell EYE – 1.3.3
PixelYourSite PRO: by PixelYourSite – 6.0.8
Print, PDF & Email by PrintFriendly: by Print
PDF
& Email by PrintFriendly – 3.7.5

Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 2.2.6
Resize Image After Upload: by iamphilrae – 1.7.2
SearchWP WooCommerce Integration: by SearchWP
LLC – 1.1.13

SearchWP: by SearchWP
LLC – 2.8.9

Slideshow: by StefanBoonstra – 2.3.1
Social Sharing Buttons By AddShoppers: by AddShoppers – 1.6.2
Sucuri Security - Auditing, Malware Scanner and Hardening: by Sucuri
Inc – 1.8.3

SyntaxHighlighter Evolved: by Alex Mills (Viper007Bond) – 3.2.1
UpdraftPlus - Backup/Restore: by UpdraftPlus.Com
DavidAnderson – 2.13.1.21

User Role Editor: by Vladimir Garagulya – 4.32.3
Aelia Foundation Classes for WooCommerce: by Aelia – 1.8.6.170405
Aelia - Google Product Feeds Multi-currency: by Aelia – 1.0.1.160412
WooChimp: by RightPress – 2.2.4
WooCommerce Address Validation: by SkyVerge – 2.1.0
Aelia Currency Switcher for WooCommerce: by Aelia – 4.4.16.170424
WooCommerce Brands: by WooThemes – 1.5.0 – 1.5.1 is available
WooCommerce Bulk Stock Management: by WooCommerce – 2.2.7 – 2.2.8 is available
WooCommerce Cart Reports: by WPBackOffice – 1.1.13
WooCommerce EU VAT Number: by WooCommerce – 2.2.0
WooCommerce - Facebook Tab: by WooThemes – 1.2.0
Follow-Up Emails: by WooCommerce – 4.4.19
WooCommerce Amazon Pay Gateway: by WooCommerce – 1.7.1
WooCommerce Stripe Gateway: by WooCommerce – 3.1.8
WooCommerce Google Analytics Integration: by WooCommerce – 1.4.2
WooCommerce Google Trusted Store: by Setubridge Technolabs – 1.0.5
WooCommerce Linnworks Integration: by VanboDevelops – 1.5.5
WooCommerce PDF Invoices: by Bas Elbers – 2.9.0
WooCommerce Google Product Feed: by Lee Willis – 7.1.1
WooCommerce Product Search: by itthinx – 1.9.0
WooCommerce Product Stock Alert: by WC Marketplace
The Grey Parrots – 1.4.3

WooCommerce Quantity Increment: by Automattic
WooThemes – 1.1.0

WooCommerce Review for Discount: by WooCommerce – 1.6.3
WooCommerce Sequential Order Numbers: by SkyVerge – 1.8.1
WooCommerce - Store Exporter Deluxe: by Visser Labs – 2.3.3
WooCommerce Subscriptions: by Prospress Inc. – 2.2.6
WooCommerce Table Rate Shipping: by WooCommerce – 3.0.3
Aelia Tax Display by Country for WooCommerce: by Aelia – 1.9.4.170415
WooCommerce Terms & Conditions Popup: by Patrick Rauland – 1.0.3
WooCommerce: by Automattic – 3.0.7
WooDojo: by WooThemes – 1.5.4
WooSidebars: by WooThemes – 1.4.3
WooCommerce Helper: by WooCommerce – 1.7.2 – Network enabled
WordPress Importer: by wordpressdotorg – 0.6.3
Yoast SEO Premium: by Team Yoast – 4.7.1
WP Crontrol: by John Blackbourn & contributors – 1.5
WP-Discourse: by Discourse – 1.3.5
WP-Mail-SMTP: by Callum Macdonald – 0.10.1
WP Post to PDF Enhanced: by Lewis Rosenthal – 1.1.1
WP Rocket: by WP Media – 2.9.11
Yoast SEO: Local: by Team Yoast and Arjan Snaterse – 4.6
Yoast SEO: News: by Team Yoast – 4.6
Yoast SEO: Video: by Team Yoast – 4.6
Yoast SEO: WooCommerce: by Team Yoast – 4.6
YOURLS Link Creator: by Andrew Norcross – 2.1.1

### Settings ###

API Enabled: ✔
Force SSL: ✔
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Shop base: #5 - /shop/
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My account: #8 - /my-account/

### Theme ###

Name: Mystile Child
Version: 1.3.17
Author URL: http://www.woothemes.com
Child Theme: ✔
Parent Theme Name: Mystile
Parent Theme Version: 1.3.19
Parent Theme Author URL: http://www.woothemes.com
WooCommerce Support: ✔

### Templates ###

Overrides: mystile-child/woocommerce/checkout/payment.php
mystile-child/woocommerce/checkout/terms.php


### subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live

### Templates ###

Scheduled Export to FTP: ✔
Scheduled Export to SFTP: ❌ The required SFTP functions are not available
```

**Comment by kcppdevelopers** [comment]
@shawaj It seems you using PayPal for WooCommerce 1.3.3 version, it is not compatible with WC Version: 3.0.7 Can you please upgrade PayPal for WooCommerce plugin and let me know if any.

**Comment by shawaj** [comment]
@kcppdevelopers - I will be happy to upgrade, but I upgraded to 1.4.0 and it broke the implementation?

Thanks

**Comment by shawaj** [comment]
Have updated, but now getting this weird tickbox:

![image](https://cloud.githubusercontent.com/assets/3359418/26201555/7c658634-3bcb-11e7-9ef4-5dbd8db080c0.png)

![image](https://cloud.githubusercontent.com/assets/3359418/26201563/879237dc-3bcb-11e7-95bc-be3d80a59048.png)


Looks ok in chekcout, but in cart it is very weird.

Perhaps this could be something they could tick on the confirm page instead of here? Can we have the payment method auto-save perhaps?

**Comment by kcppdevelopers** [comment]
@shawaj PayPal for WooCoomerce 1.4.4 version is now official available. Can you upgrade to 1.4.4 version and let me know if any.

**Comment by shawaj** [comment]
Yes I've upgraded to that and still getting the issue with the tick boxes above?

**Comment by angelleye** [comment]
@shawaj Are you referring to the option to Save PayPal account for future use?  If so, that can be removed by disabling the Token Payments option in the settings.

**Comment by kcppdevelopers** [comment]
@angelleye   I have resolved checkbox CSS related issue: https://www.screencast.com/t/5zR0qYehh8FC

**Comment by angelleye** [comment]
@shawaj This has been resolved in the release branch and will be included in 1.4.5 that we'll be pushing before long.
